### DISCUSSION TITLE
Stream example

### DISCUSSION THREAD

**Discussion by Touville01** [roboflow/supervision#1231] [Q&A] [answered]
Hi,

1. I was wondering if Supervision provides stream analysis for the following: -Count people in zone, Heatmap and tracking, tracking and speed. For the moment it's only available for time_in_zone. Is it similar for these example ? 

Also, so far, I used your ```stream_from_file.py``` script to generate the RTSP address, but  I would like to integrate my own camera into it.

2. Could you please advise on the best way to proceed with this? Specifically:

Camera Integration: What is the best approach to integrate my own camera while sticking closely to the provided examples? Should I generate an RTSP address from the camera's feed, or is there a better method?

Technical Requirements: What are the prerequisites I need to consider in terms of resolution, frames per second (FPS), and other technical aspects to ensure optimal performance with YOLOv8s?

Configuration: Are there any specific configurations or settings in the Supervision scripts ?

Thank you!

**Comment by SkalskiP** [comment, marked as answer]
Hi @Touville01 👋🏻,

1. Unfortunately, we don't have stream examples for the projects you mentioned.
2. You need a streaming camera in your network using the RTSP protocol.
3. As mentioned in the tutorial, if you plan on tracking, it's better to provide a stream with a higher FPS.

**Comment by Touville01** [comment]
Thank you !
If there are challenges, is it possible to do easily real-time counting by zone? Is it more complex than, for example tracking time spent in zones in ultralytics(inference)_stream_example.py code ?

**Reply by LinasKo** [reply]
Hi @Touville01,

We have a tutorial on counting objects in zones. I've just opened a PR to adapt it to latest `supervision` version.

You may take a look here: https://github.com/LinasKo/notebooks/blob/7447f56898748e877a5afb26775545266d3ac191/notebooks/how-to-detect-and-count-objects-in-polygon-zone.ipynb
